collector/logical_router_port_collector.go

```diff
@@ -26,8 +26,18 @@ type logicalRouterPortCollector struct {
 	txTotalByte     *prometheus.Desc
 }
 
-func newLogicalRouterPortCollector(apiClient *nsxt.APIClient, logger log.Logger) prometheus.Collector {
+type logicalRouterPortStatisticMetric struct {
+	LogicalRouterPort manager.LogicalRouterPort
+	Rx                *manager.LogicalRouterPortCounters
+	Tx                *manager.LogicalRouterPortCounters
```

Comment:
nit: should do the golang (or networking?) way
```suggestion
	TX                *manager.LogicalRouterPortCounters
```